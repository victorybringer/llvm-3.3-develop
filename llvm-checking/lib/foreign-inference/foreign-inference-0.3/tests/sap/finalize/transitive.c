#include <stdlib.h>

struct S {
  int *p1;
  int *p2;
};

void g(struct S *s) {
  free(s->p2);
}

void f(struct S *s) {
  g(s);
}
