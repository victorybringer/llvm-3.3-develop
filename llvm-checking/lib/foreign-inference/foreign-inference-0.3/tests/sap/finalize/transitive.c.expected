fromList [ (("g", "s"), fromList [("%struct.S*", [AccessDeref,AccessField 1])])
         , (("f", "s"), fromList [("%struct.S*", [AccessDeref,AccessField 1])])
         ]
